mogwai/src/view/dom.rs

```diff
@@ -139,7 +140,8 @@ impl Drop for ViewInternals {
             }
             // TODO: Investigate whether we can drop rx responders before the element Rc
             // count drops to 1
-            self.rxs.iter().for_each(|rx| rx.drop_responder());
+            // TODO: Do responders need to explicitly be dropped?
+            // self.rxs.iter().for_each(|rx| rx.drop_responder());
```

Comment:
This made the example work but I don't know what bad things this does if responders are not explicitly dropped.

Reply:
Interesting! So you had to comment this out for the example to work? In that case - this will make a nice unit test.